labgrid/remote/client.py

```diff
@@ -664,6 +664,9 @@ def _get_target(self, place):
                     from .. import StepReporter
                     StepReporter.start()
                 strategy = target.get_driver("Strategy")
+                if self.args.initial_state:
+                    print(f"Setting initial state to {self.args.initial_state}")
+                    strategy.force(self.args.initial_state)
```

Comment:
We should check and error out with a warning if the strategy doesn't implement the `force()` method.